packages/core/src/core.ts

```diff
@@ -114,3 +115,21 @@ export const CORE_MIXIN_INDICES = 'mixin:core.Indices' as Ref<Mixin<Indices>>
 export interface AnyLayout {
   [key: string]: PropertyType
 }
+
+//
+
+export interface Tx extends Doc {
+  _date: DateProperty
```

Comment:
Is it still relevant to use '_'?

Reply:
no :) I just preserved it for "compatibility", afraid to break more things if remove '_'

Reply:
maybe later I'll do cleanup job to remove '_'s everywhere without introducing other changes 😄